.expeditor/verify.pipeline.yml

```diff
@@ -0,0 +1,62 @@
+---
+expeditor:
+  cached_folders:
+    - vendor
+  defaults:
+    buildkite:
+      retry:
+        automatic:
+          limit: 1
+      timeout_in_minutes: 30
+
+steps:
+
+- label: run-lint-and-specs-ruby-2.5
+  command:
+    - .expeditor/run_linux_tests.sh rake
+  expeditor:
+    executor:
+      docker:
+        image: ruby:2.5
+
+- label: run-lint-and-specs-ruby-2.6
+  command:
+    - .expeditor/run_linux_tests.sh rake
+  expeditor:
+    executor:
+      docker:
+        image: ruby:2.6
+
+- label: run-lint-and-specs-ruby-2.7
+  command:
+    - .expeditor/run_linux_tests.sh rake
+  expeditor:
+    executor:
+      docker:
+        image: ruby:2.7
+
+- label: run-lint-and-specs-ruby-3.0
+  command:
+    - .expeditor/run_linux_tests.sh rake
+  expeditor:
+    executor:
+      docker:
+        image: ruby:3.0
+
+- label: run-lint-and-specs-ruby-3.1
+  command:
+    - .expeditor/run_linux_tests.sh rake
+  expeditor:
+    executor:
+      docker:
+        image: ruby:3.1
+
+- label: run-specs-windows
+  command:
+    - bundle config --local path vendor/bundle
+    - bundle install --jobs=7 --retry=3
+    - bundle exec rake
+  expeditor:
+    executor:
+      docker:
+        host_os: windows
```

Comment:
Now you can use the rubydistros windows image with ruby 3.1 installed for the test on windows.

```
- label: run-specs-ruby-3.1-windows
  command:
    - bundle config --local path vendor/bundle
    - bundle install --jobs=7 --retry=3
    - bundle exec rake
  expeditor:
    executor:
      docker:
        host_os: windows
        shell: ["powershell", "-Command"]
        image: rubydistros/windows-2019:3.1
```